build/build/src/project/engine.rs

```diff
@@ -45,29 +45,15 @@
 //         &self,
 //         context: Context,
 //         job: BuildTargetJob<Self>,
-//     ) -> BoxFuture<'static, Result<Self::Artifact>> {
-//         let WithDestination { inner, destination } = job;
-//         let this = self.clone();
-//         async move {
-//             let paths = crate::paths::Paths::new_versions(&inner.repo_root, inner.versions)?;
-//             let context = crate::engine::context::RunContext {
-//                 operation: crate::engine::Operation::Build,
-//                 goodies: GoodieDatabase::new()?,
-//                 config: BuildConfigurationFlags {
-//                     clean_repo: false,
-//                     build_engine_package: true,
-//                     ..crate::engine::NIGHTLY
-//                 }
-//                 .into(),
-//                 inner: context,
-//                 paths,
-//             };
-//             let artifacts = context.build().await?;
-//             let engine_distribution =
-//                 artifacts.packages.engine.context("Missing Engine Distribution!")?;
-//             ide_ci::fs::mirror_directory(&engine_distribution.dir, &destination).await?;
-//             this.adapt_artifact(destination).await
-//         }
-//         .boxed()
+//     ) -> BoxFuture<'static, Result<Self::Artifact>> { let WithDestination { inner, destination }
+//       = job; let this = self.clone(); async move { let paths =
+//       crate::paths::Paths::new_versions(&inner.repo_root, inner.versions)?; let context =
+//       crate::engine::context::RunContext { operation: crate::engine::Operation::Build, goodies:
+//       GoodieDatabase::new()?, config: BuildConfigurationFlags { clean_repo: false,
+//       build_engine_package: true, ..crate::engine::NIGHTLY } .into(), inner: context, paths, };
+//       let artifacts = context.build().await?; let engine_distribution =
+//       artifacts.packages.engine.context("Missing Engine Distribution!")?;
+//       ide_ci::fs::mirror_directory(&engine_distribution.dir, &destination).await?;
+//       this.adapt_artifact(destination).await } .boxed()
```

Comment:
this is unformatted. The previous version was much nicer.

Reply:
Also the question is: why do we keep entirely-commented file in our repo? @mwu-tow ?

Reply:
Obviously I wasn't touching commented code, it must be rustfmt doing that. I also don't understand why we have commented code in the repo, I was always told on reviews to remove it.